src/components/Markdown.tsx

```diff
@@ -1,46 +1,29 @@
-import ReactMarkdown from "react-markdown";
+import ReactMarkdown, { PluggableList } from "react-markdown";
 import remarkGfm from "remark-gfm";
 import rehypeRaw from "rehype-raw";
-import rehypeSanitize, { Options } from "rehype-sanitize";
-import { useMemo } from "react";
-import { PluggableList } from "unified";
+import rehypeSanitize from "rehype-sanitize";
+
+// The Remark and Rehype plugins to use in conjunction with ReactMarkdown.
+const unifiedPlugins: { remark: PluggableList; rehype: PluggableList } = {
+  remark: [remarkGfm],
+  rehype: [rehypeRaw, rehypeSanitize],
```

Comment:
Can we add a test to check that HTML is rendered/sanitized correctly?

Reply:
I did try to add such a test. However, it seems that `react/testing-library` does some degree of sanitization automatically. Even without the `rehype-sanitize` plugin, the DOM I rendered through the `testing-library` was free of invalid/malicious HTML. In Storybook, though, I manually confirmed that malicious HTML appeared without `rehype-sanitize` and was filtered out when that plugin was added back in.